include/sys/vmem.h

```diff
@@ -19,7 +20,7 @@ void init_vmem(void);
 
 /*! \brief Create a new vmem arena.
  * You need to specify quantum, the smallest unit of allocation. */
-vmem_t *vmem_create(const char *name, vmem_size_t quantum);
+vmem_t *vmem_create(const char *name, vmem_size_t quantum, bool grow);
```

Comment:
I have a feeling @cahirwpz won't like this and would prefer the third parameter to be `vmem_flags_t` instead.

Reply:
done